### ISSUE TITLE
Detecting LDAP user requires password change (and informing the user)

### ISSUE THREAD

**Issue by ruckc** [FusionAuth/fusionauth-issues#992]
### Problem
FA should detect if an LDAP user requires a password change to login and inform the user to create a seamless SSO experience.

### Solution
When an LDAP Error 49 occurs, it should inspect the error (or query the ldap server) for fields that inform the user that their password must be changed to complete the login process, instead of just responding with a generic invalid username/password failure.

### Alternatives/workarounds
None

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by robotdan** [comment]
Is this a duplicate of https://github.com/FusionAuth/fusionauth-issues/issues/895 ?

**Comment by ruckc** [comment]
No, #895 looks to be part of the migration workflow, we don't want to migrate users, but have them remain in LDAP for authentication, so we want FusionAuth to detect the ldap error and in #993 have it let them change their password in the LDAP.
